### PR TITLE
updated functionality and strings for OIDC

### LINKED ISSUE #5767: add 'powered by kolibri' option to theme
### Observed behavior

in support of UN Women

### Expected behavior

…

### User-facing consequences

…

### Errors and logs

…

### Steps to reproduce

…

### Context

…

### LINKED ISSUE #5768: update privacy modal with SSO-related material
### Observed behavior

In support of UN Women integration

### Expected behavior

…

### User-facing consequences

…

### Errors and logs

…

### Steps to reproduce

…

### Context

…

### LINKED ISSUE #5799: add support for OIDC in account creation flow
### Observed behavior

currently SSO works for sign in but not account creation

### Expected behavior

the 'next' parameter needs to be passed from the sign in page to the account creation page and handled there

### User-facing consequences

The following flow is not supported:

> 1. User comes to SCE portal and click on login for one server, system would redirect to Kolibri portal.
> 2. On Kolibri portal, user decides to not login but register and clicks on register.
> 3. After registration, user is redirecting to Kolibri portal instead of returning back to SCE portal.

### Context

0.12.5

### PR DESCRIPTION
**Pull request by indirectlylit**
### Summary

* New "powered by" link to LE website, shown when theme enables it
* New "What's this?" link which opens popup, shown when OIDC flow is in progress
  * NOTE: currently uses "generic" info, not something with the name (SCE Portal) in it
* Now passes the `next` parameter through the account creation flow
* Updates privacy info modal with OIDC info when plugin is active

Adding new strings and behaviors to support SSO using OIDC for UN Women

| description | screenshot |
|--|--|
| Normal login screen | image |
| SSO login | image  |
| "Powered by" popup shown only during SSO flow | image |
| Privacy info update | image |

### Reviewer guidance

Used by https://github.com/learningequality/kolibri-un-women-common-plugin/pull/5

### References

* fixes #5767
* fixes #5768
* fixes #5799

----

### Contributor Checklist

PR process:

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [x] Contributor has fully tested the PR manually
- [x] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

**Comment by indirectlylit** [comment]
updates made based on feedback from @carinediaz:

* No more "What's this?" link
* During SSO flow, 'powered by' link opens the 'what's this' popup instead of going directly to website
* Added link to website from 'what's this?' popup

File: kolibri/core/assets/src/views/PrivacyInfoModal.vue [hunk 1]
```diff
@@ -109,6 +119,9 @@
         'Unlike many online web services that are similarly accessed through a web browser, there are thousands of independent Kolibri installations around the world – including this one. Each Kolibri installation is managed and controlled by the owner of the device that it is installed on.',
       kolibriAboutP4:
         'In order to improve the quality of Kolibri and the content on it, Learning Equality might collect anonymized usage information when Kolibri has access to the internet. This may include IP addresses associated with the server, device details such as the operating system and time zone, and aggregate statistics about the users and content.',
+      openIdH1: 'Signing in to third-party applications using Kolibri',
+      openIdP1:
+        'It is possible to use Kolibri to register or sign in to third-party applications. If you do this, the other application will have access to your Kolibri username, unique user ID, and full name.',
```
Comment (jredrejo) [inline kolibri/core/assets/src/views/PrivacyInfoModal.vue:124]: This works fine, but the strings appear in English, we'll have to translate them and recreate the cloud installation

File: kolibri/plugins/user/assets/src/views/SignUpPage.vue [hunk 2]
```diff
@@ -236,6 +236,14 @@
           !this.facilityIsInvalid
         );
       },
+      nextParam() {
+        // query is after hash
+        if (this.$route.query.next) {
+          return this.$route.query.next;
+        }
+        // query is before hash
+        return getUrlParameter('next');
```
Comment (jredrejo) [inline kolibri/plugins/user/assets/src/views/SignUpPage.vue:246]: This produces an error, this functions needs to be imported from SignInPage/index.vue or added to this file

**Review by jredrejo — DISMISSED** [review]
Only the `getUrlParameter` issue must be fixed. Other than that everything looks/works fine.

File: kolibri/core/assets/src/views/PrivacyInfoModal.vue (hunk 1 above)
Comment (indirectlylit) [reply kolibri/core/assets/src/views/PrivacyInfoModal.vue:124]: yes, we're waiting on translations and should have them by EOD thursday

File: kolibri/plugins/user/assets/src/views/SignUpPage.vue (hunk 2 above)
Comment (indirectlylit) [reply kolibri/plugins/user/assets/src/views/SignUpPage.vue:246]: thanks good catch

Comment (indirectlylit) [reply kolibri/plugins/user/assets/src/views/SignUpPage.vue:246]: pushed fix

File: kolibri/plugins/user/assets/src/views/SignInPage/index.vue [hunk 3]
```diff
@@ -157,24 +198,21 @@
...
   import KTextbox from 'kolibri.coreVue.components.KTextbox';
+  import KModal from 'kolibri.coreVue.components.KModal';
   import CoreLogo from 'kolibri.coreVue.components.CoreLogo';
   import { validateUsername } from 'kolibri.utils.validators';
   import UiAutocompleteSuggestion from 'keen-ui/src/UiAutocompleteSuggestion';
   import PrivacyInfoModal from 'kolibri.coreVue.components.PrivacyInfoModal';
   import UiAlert from 'keen-ui/src/UiAlert';
   import responsiveWindow from 'kolibri.coreVue.mixins.responsiveWindow';
   import urls from 'kolibri.urls';
+  import { crossComponentTranslator } from 'kolibri.utils.i18n';
```
Comment (rtibbles) [inline kolibri/plugins/user/assets/src/views/SignInPage/index.vue:209]: Booooo!

**Review by rtibbles — APPROVED** [review]
Seems fine - I look forward to the day cross component translator is done for!

**Pull request merged**